src/main/java/uk/gov/hmcts/reform/et/model/service/hearingvalues/ServiceHearingValues.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.hmcts.reform.et.model.service.hearingvalues;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.et.common.model.hmc.CaseCategory;
+import uk.gov.hmcts.et.common.model.hmc.CaseFlags;
+import uk.gov.hmcts.et.common.model.hmc.HearingLocation;
+import uk.gov.hmcts.et.common.model.hmc.HearingWindow;
+import uk.gov.hmcts.et.common.model.hmc.Judiciary;
+import uk.gov.hmcts.et.common.model.hmc.PanelRequirements;
+import uk.gov.hmcts.et.common.model.hmc.PartyDetails;
+import uk.gov.hmcts.et.common.model.hmc.ScreenNavigation;
+import uk.gov.hmcts.et.common.model.hmc.Vocabulary;
+
+import java.util.List;
+
+@SuppressWarnings("PMD.TooManyFields")
+@Data
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class ServiceHearingValues {
+
+    private String hmctsServiceID;
```

Comment:
How come we are only setting @JsonProperty for some but not all of them?

Reply:
I have removed the unneeded json properties, there are now only used when a property is being converted to/from camel case.